### ISSUE TITLE
Want to hard code a default email address

### ISSUE THREAD

**Issue by monodrone** [PHPMailer/PHPMailer#1821]
# Problem description
I want to hard code a default email address (other options are working fine except addAddress function )
$mail->addAddress('user@gmail.com', 'user');
     in PHPMailer.php 

I am planing to use an HTML tag <form action=mailsender.php > REST name elements and collect 
 `$_POST["Email_Subject"];` andother non sensitive form data to use with mail object vars.  The made object instance of mail or PHPMailer.php file will sit under php server user and will not be exposed while web exposed files are stored under WWW user for the public (mailsender.php ).
Simply put, the objective is to have user/pass stored in mailsender.php under WWW user not be downloaded
with a directory traversal. Not sure where my logic fails...

Can I do an array default value ? It should inherit default email.
# Code to reproduce
   Original the variables were array initialization [ ];
 
    protected $to = array(0=> 'user@gmail.com');
    protected $cc = array(0=> 'user@gmail.com');
    protected $bcc = array(0=> 'user@gmail.com');

# Debug output

ERROR: RCPT TO command failed: 553 5.1.2 The recipient address <p> is not a valid RFC-5321 address.

**Comment by Synchro** [comment]
This isn't a good approach - don't ever edit library files. Create a subclass instead and add your hard-coded addresses and override default settings in there:

```php
namespace myApp;
use PHPMailer\PHPMailer\PHPMailer;
require 'vendor/autoload.php';
class myMailer extends PHPMailer
{
    public $Host = 'mail.example.com';
    public $Port = 587;
    public $SMTPSecure = 'tls';
    public $Username = 'username';
    public $Password = 'Password1';
    public function __construct()
    {
        parent::__construct();
        $this->addAddress('user@gmail.com', 'user');
        $this->addBCC('user2@example.com', 'user 2');
    }
}
```

Then use your own class when creating an instance:

```php
$mail = new myApp\myMailer();
```

Also, what you say isn't usually a problem - your code should be outside (usually 1 level above) your web root anyway. Typically you have a `public` folder which contains your static assets (CSS, JS, images), and a single `index.php` that kicks off your app and loads code from `../`.
